.github/workflows/main.yml

```diff
@@ -7,14 +7,12 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        go: [ '1.10', '1.11', '1.12' ]
+        go: [ '1.15', '1.16.4' ]
```

Comment:
any reason to explicitly call out the patch version in `1.16.4`?

Reply:
none, aside from it's what I use locally. I'll switch it to just 1.16